### PR TITLE
Pagination of organisations

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

File: app/controllers/organisations_controller.rb
Comment (mermop) [inline app/controllers/organisations_controller.rb:5]: it's failing on `policy_scope` here

**Review by mermop — APPROVED** [review]
good to merge once tests pass

**Comment by Br3nda** [comment]
should pass after we merge #172

**Pull request merged**
